src/patternfly/components/Page/examples/Page.md

```diff
@@ -173,7 +173,7 @@ This component provides the basic chrome for a page, including sidebar, header,
 | `.pf-m-light` | `.pf-c-page__main-section` | Modifies a main page section to have a light theme. |
 | `.pf-m-dark-200` | `.pf-c-page__main-section` |  Modifies a main page section to have a dark theme and a dark transparent background. |
 | `.pf-m-dark-100` | `.pf-c-page__main-section` |  Modifies a main page section to have a dark theme and a darker transparent background. |
-| `.pf-m-no-padding` | `.pf-c-page__main-section` | Modifies a main page section to have no padding. |
-| `.pf-m-no-padding-mobile` | `.pf-c-page__main-section` | Modifies a main page section to have no padding on mobile. |
+| `.pf-m-no-padding`, `.pf-m-no-padding-{-on-[breakpoint]}` | `.pf-c-page__main-section` | Removes padding from the main page section at an optional breakpoint |
+| `.pf-m-padding-{-on-[breakpoint]}` | `.pf-c-page__main-section` | Modifies the main page section to have padding at an optional breakpoint. |
```

Comment:
The SCSS creates `.pf-m-padding`, though it won't currently be used since it has padding by default. Probably best if we indicate it's there in the selector with this change. Also since the docs say "at an optional breakpoint," meaning the breakpoint isn't a required part of the selector.

```suggestion
| `.pf-m-padding{-on-[breakpoint]}` | `.pf-c-page__main-section` | Modifies the main page section to have padding at an optional breakpoint. |
```